wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -176,90 +179,206 @@ Example of an inherited Timing Point:
 
 `10171,-100,4,2,0,60,0,1`
 
-### Colours
+Colours
+-------
 
 Combo# (Integer List) is a list of three numbers, each from 0 - 255 defining an RGB color.
 
 `Combo1 : 245,245,245`
 
-### Hit Objects
+Hit Objects
+-----------
 
-**Hit Circle Syntax:**
+### Common structure
 
-`x,y,time,type,hitSound,addition`
+This section is made of CSV lines. The first 5 fields are common to every hit objects, and an optional last *addition* fields.
 
-`164,260,2434,1,0,0:0:0:0:`
+**Syntax**: `x,y,time,type,hitSound...,addition`
 
-A hit circle is a single hit in all osu! game modes.
+#### Position
+
+*x* and *y* are integers representing the position of the center of the hit object. *x* ranges from 0 to 512 pixels, inclusive, and *y* ranges from 0 to 384 pixels, inclusive. The origin, (0, 0) is at the top left of the screen.
+
+To map these coordinates for a standard 640x480 screen, you need to add 64 pixels to *x* and 48 pixels to *y* to respect a uniform padding. Without the padding, an object at (0, 0) will be cut on the top left for the screen.
+
+Hit objects whose position is irrelevant, like spinners in osu! or any note in osu!taiko, have the special position (256,192), which is the center of the screen. For osu!mania, only *x* is relevant.
```

Comment:
maybe note that the position doesn't actually matter either. that value is just the default set by the osu! editor

Reply:
Reformulated.